lib/widgets/app.dart

```diff
@@ -209,11 +208,10 @@ class _ZulipAppState extends State<ZulipApp> with WidgetsBindingObserver {
 
           onGenerateInitialRoutes: (_) {
             return [
-              MaterialWidgetRoute(page: const ChooseAccountPage()),
-              if (initialAccountId != null) ...[
+              if (initialAccountId != null)
                 HomePage.buildRoute(accountId: initialAccountId),
-                InboxPage.buildRoute(accountId: initialAccountId),
-              ],
+              if (initialAccountId == null)
+                MaterialWidgetRoute(page: const ChooseAccountPage()),
```

Comment:
nit: put the no-account case first; it's the case that always happens first chronologically, and it's also the unusual case so going first is consistent with our usual early-return structure